core/extension/extension_api_dump.cpp

```diff
@@ -458,6 +458,7 @@ Dictionary GDExtensionAPIDump::generate_extension_api() {
 		// Global enums and constants.
 		Array constants;
 		HashMap<String, List<Pair<String, int64_t>>> enum_list;
+		HashMap<String, bool> enum_is_bitfield;
```

Comment:
Alternatively, we could create a struct like:

```c++
struct EnumInterface {
	String name;
	bool is_bitfield;
	List<Pair<String, int64_t>> values;
};
```

Then we'd only need one map:

```c++
HashMap<String, EnumInterface> enum_list;
```

Reply:
I'd rather stick to the current solution, it is in theory less performant (two time more hashmap insert/lookups) but this code is not performance critical so it's no big deal. On the other hand avoiding a new struct declaration make the code a bit simpler (no need to jump between struct declaration and code using it).

But that's not a very strong take, if you really prefere it I will make the change @raulsntos  😃

Reply:
I'm fine either way so whatever is preferred by the team.